pkgs/orf/gping/pkg.yaml

```diff
@@ -0,0 +1,2 @@
+packages:
+- name: orf/gping@gping-v1.3.1
```

Comment:
📝 To update this tool with Renovate, we'll have to update aqua-renovate-config.
https://github.com/aquaproj/aqua-renovate-config/blob/main/default.json

Reply:
https://github.com/aquaproj/aqua-renovate-config/pull/121